test/unit/uk/gov/hmrc/exports/services/mapping/goodsshipment/consignment/ConsignmentBuilderSpec.scala

```diff
@@ -27,24 +27,28 @@ import util.testdata.ExportsDeclarationBuilder
 import wco.datamodel.wco.dec_dms._2.Declaration
 import wco.datamodel.wco.dec_dms._2.Declaration.GoodsShipment
 
-class ConsignmentBuilderSpec extends WordSpec with Matchers with ExportsDeclarationBuilder with MockitoSugar {
+class ConsignmentBuilderSpec extends WordSpec with Matchers with ExportsDeclarationBuilder with MockitoSugar with BeforeAndAfterEach {
 
-  private val mockContainerCodeBuilder = mock[ContainerCodeBuilder]
-  private val mockGoodsLocationBuilder = mock[GoodsLocationBuilder]
-  private val mockDepartureTransportMeansBuilder = mock[DepartureTransportMeansBuilder]
-  private val mockTransportEquipmentBuilder = mock[TransportEquipmentBuilder]
+  private val containerCodeBuilder = mock[ContainerCodeBuilder]
+  private val goodsLocationBuilder = mock[GoodsLocationBuilder]
+  private val departureTransportMeansBuilder = mock[DepartureTransportMeansBuilder]
+  private val transportEquipmentBuilder = mock[TransportEquipmentBuilder]
 
-  private val builder =
-    new ConsignmentBuilder(mockGoodsLocationBuilder, mockContainerCodeBuilder, mockDepartureTransportMeansBuilder, mockTransportEquipmentBuilder)
+  private val builder = new ConsignmentBuilder(goodsLocationBuilder, containerCodeBuilder, departureTransportMeansBuilder, transportEquipmentBuilder)
+
+  override protected def beforeEach(): Unit = {
+    super.beforeEach()
+    reset(containerCodeBuilder, goodsLocationBuilder, departureTransportMeansBuilder, transportEquipmentBuilder)
+  }
```

Comment:
I think this should be in `afterEach`. Resetting should be after test. Of course if you run it it doesn't make any difference, but IMO this is more logically if I use something and after this I'm resetting :)